skactiveml/pool/_bald.py

```diff
@@ -0,0 +1,547 @@
+import numpy as np
```

Comment:
"""
Code is based on https://blackhc.github.io/batchbald_redux/ distributed under the Apache-2.0 license.
"""

Reply:
Rename file `batch_bald.py`